public/app/plugins/datasource/influxdb/components/ConfigEditor.tsx

```diff
@@ -289,7 +267,60 @@ export class ConfigEditor extends PureComponent<Props> {
           </div>
         </div>
 
-        {options.jsonData.version === InfluxVersion.Flux ? this.renderInflux2x() : this.renderInflux1x()}
+        {options.jsonData.version === InfluxVersion.Flux && (
+          <InfoBox>
+            <h5>Support for flux in Grafana is currently in beta</h5>
+            <p>
+              Please report any issues to: <br />
+              <a href="https://github.com/grafana/grafana/issues/new/choose">
+                https://github.com/grafana/grafana/issues
+              </a>
+            </p>
+          </InfoBox>
+        )}
+
+        <DataSourceHttpSettings
+          showAccessOptions={true}
+          dataSourceConfig={options}
+          defaultUrl="http://localhost:8086"
+          onChange={onOptionsChange}
+        />
+
+        <div className="gf-form-group">
+          <div>
+            <h3 className="page-heading">InfluxDB Details</h3>
+          </div>
+          {options.jsonData.version === InfluxVersion.Flux ? this.renderInflux2x() : this.renderInflux1x()}
+          <div className="gf-form-inline">
+            <div className="gf-form">
+              <InlineFormLabel
+                tooltip="Max number of series that will be returned from the data source query. Defaults to 1000."
```

Comment:
I'd make this more insightful: `Limit the number of series/tables that Grafana will process. Lower this number to prevent abuse, and increase it if you have lots of small time series and not all are shown. Defaults to 1000.`